### PR TITLE
[feat](Nereids) add restore state check of partition after rewrite

### PR DESCRIPTION
**Pull request by LiBinfeng-01**
after nereids rewrite, it can not be partition with restore state, if so, throw an exception

### CODE REVIEW

**Comment by LiBinfeng-01** [comment]
run buildall

**Comment by LiBinfeng-01** [comment]
run p0

**Comment by LiBinfeng-01** [comment]
run cloud_p0

**Comment by LiBinfeng-01** [comment]
run p0

**Comment by LiBinfeng-01** [comment]
run cloud_p0

**Comment by morrySnow** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/cloud/rpc/MetaServiceClient.java [hunk 1]
```diff
@@ -79,7 +79,7 @@ private long connectionAgeExpiredAt() {
         if (!isMetaServiceEndpointList && connectionAgeBase > 1) {
             long base = TimeUnit.MINUTES.toMillis(connectionAgeBase);
             long now = System.currentTimeMillis();
-            long rand = random.nextLong(base);
+            long rand = 0;
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/cloud/rpc/MetaServiceClient.java:82]: ?

**Comment by LiBinfeng-01** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

**Review by englefly — APPROVED** [review]

**Pull request merged**